pkg/route/api/helper.go

```diff
@@ -20,11 +20,16 @@ func RouteLessThan(route1, route2 *Route) bool {
 	if route1.CreationTimestamp.Before(route2.CreationTimestamp) {
 		return true
 	}
-	if route1.CreationTimestamp == route2.CreationTimestamp && route1.UID < route2.UID {
-		return true
-	}
-	if route1.Namespace < route2.Namespace {
-		return true
+
+	if route1.CreationTimestamp == route2.CreationTimestamp {
+		if route1.UID < route2.UID {
+			return true
+		}
+		if route1.Namespace < route2.Namespace {
```

Comment:
The change was done in April - so should be 1.3 as well. 
https://github.com/openshift/origin/commit/ee4035a3e4e45d3a6e2a4d8885e3ddcb007fdf9c
@marun can confirm. 

And did you mean backport on 1.3 on origin or OSE or both?